playbooks/kubevirt-resources.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: localhost
+  connection: local
+  gather_facts: False
+  # unset http_proxy. required for running in the CI
+  environment:
```

Comment:
We don't need the environment key here (the login is done in a different playbook).

Reply:
When I removed it it would break CI on some weird error.